knowledge_graph/network_class.py

```diff
@@ -107,28 +91,29 @@ def dfs_for_classes(self, node):
         if classes:
 
             for ont_class in classes:
-                if ont_class != owl.Thing:
+                # if ont_class != owl.Thing:
+                if isinstance(ont_class, owlready2.entity.ThingClass):
                     self.add_class_to_explore(ont_class)
 
             while self.class_family:
-                parent2, children2, edge_type2 = self.class_family[-1]
-                visited_classes.add(parent2)  # these are not all classses
-
-                try:
-                    child2 = next(children2)
-                    if child2 != owl.Thing:
-
-                        if child2 in self.ontology.individuals():
-                            self.add_child_to_result(child2, node, edge_type2)
-                        elif (
-                            child2 not in visited_classes
-                            and child2 in self.ontology.classes()
-                        ):
-                            visited_classes.add(child2)
-                            self.add_class_to_explore(child2)
-
-                except StopIteration:
-                    self.class_family.pop()
+                parent2, children2, edge_type2 = self.class_family.pop()
+                visited_classes.add(parent2)
+                for child2 in children2:
+                    # if child2 != owl.Thing: # ?
+                    if child2 == owl.Thing:  # fr though, what is this?
```

Comment:
owl.Thing is the highest level class that all classes are under. When it gets to owl.Thing it can't go any further